openshift4/tusd/overlays/prod/patch.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: tusd
+spec:
+  template:
+    spec:
+      containers:
+        - name: tusd
+          image: "image-registry.openshift-image-registry.svc:5000/4c2ba9-tools/tusd:prod"
+          env:
+            - name: S3_PREFIX
+              value: mds-files-prod/
+          resources:
+            requests:
+              cpu: 10m
+              memory: 32Mi
+            limits:
+              cpu: 30m
+              memory: 96Mi
+          terminationMessagePath: /dev/termination-log
```

Comment:
Do we need to specify this path, and other constant values in every patch? Just thinking about cutting down on clutter if they don't change values.